code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -167,7 +168,7 @@
 	id = "methboom1"
 	required_temp = 380 //slightly above the meth mix time.
 	required_reagents = list(/datum/reagent/drug/methamphetamine = 1)
-	strengthdiv = 6
+	strengthdiv = 10
```

Comment:
I think this change should be reverted due to the fact that 10 is the same as pot-water, while this is so much harder to make.

Reply:
done